artifacts/ts/Add.ts

```diff
@@ -73,40 +73,48 @@ class Factory extends ContractFactory<AddInstance, AddTypes.Fields> {
 
   tests = {
     add: async (
-      params: TestContractParams<
-        AddTypes.Fields,
-        { array: [bigint, bigint] },
-        {}
+      params: Omit<
+        TestContractParams<AddTypes.Fields, { array: [bigint, bigint] }, never>,
```

Comment:
We could make `never` the default type and don't need to generate it in most cases.